data-raw/nwsl_data.R

```diff
@@ -0,0 +1,22 @@
+#code to create dataframe with NWSL data. 
+
+library(dplyr)
+
+nwsl_data <- data.frame(name = c("North Carolina Courage", "Utah Royals FC", "Houston Dash",
+                                 "Orlando Pride", "Chicago Red Stars", "Sky Blue FC", "Reign FC",
+                                 "Portland Thorns FC", "Washington Spirit"),
+                        league = rep("nwsl", 9),
```

Comment:
You don't need to use `rep()` here, because if you just put in a single value it will recycle it.